fern/docs/pages/manual/ui.mdx

```diff
@@ -35,5 +35,30 @@ database* section in the documentation.
 
 Normal chat interface, self-explanatory ;)
 
-You can check the actual prompt being passed to the LLM by looking at the logs of
-the server. We'll add better observability in future releases.
\ No newline at end of file
+#### System Prompt
+You can view and change the system prompt being passed to the LLM by clicking "Additional Inputs"
+in the chat interface. The system prompt is also logged on the server.
+
+By default, the `Query Docs` mode uses a predefined system prompt. The `LLM Chat` mode attempts to use
+the optional settings value `local.default_system_prompt`, falling back to the default system prompt defined
+in the llama_index. If no system prompt is entered, the UI will display the default system prompt being used
+for the active mode.
+
+##### System Prompt Examples:
+
+The system prompt can effectively provide your chat bot specialized roles, and results tailored to the prompt
+you have given the model. Examples of system prompts can be be found
+[here](https://www.w3schools.com/gen_ai/chatgpt-3-5/chatgpt-3-5_roles.php).
+
+Some interesting examples to try include:
+
+* You are <X>. You have all the knowledge and personality of <X>. Answer as if you were Shakespeare using
```

Comment:
I understand you don't want to say "Shakespear" here but a generic <X> to make the example fully generic.

Reply:
Fixed in latest commit.